modules/catalog/src/main/java/org/apache/ignite/internal/catalog/commands/ColumnParams.java

```diff
@@ -158,7 +158,7 @@ public Builder defaultValue(DefaultValue defaultValue) {
          *
          * @return {@code this}.
          */
-        public Builder precision(int precision) {
+        public Builder precision(Integer precision) {
```

Comment:
Please add `@Nullable` for method `org.apache.ignite.internal.catalog.commands.ColumnParams#precision` and specify in documentation what `null` means.